### PR TITLE
1923: Remove unused uberproject code

### PR DESCRIPTION
**Pull request by chrismiceli**
remove edit_uberproject.php
cleanup unused code in editproject.php
migrate tables to remove uber references

available in sandbox here:
https://www.pgdp.org/~chrismiceli/c.branch/issue/1923
I did not run the migration scripts in the sandbox environment though.
Thanks @cpeel for the script!.

### CODE REVIEW

File: SETUP/upgrade/14/20200827_remove_uber_projects.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php
+$relPath='../../../pinc/';
+include_once($relPath.'base.inc');
+
+header('Content-type: text/plain');
+
+// ------------------------------------------------------------
+
+echo "Removing uber project column from project tables...\n";
```
Comment (srjfoo) [inline SETUP/upgrade/14/20200827_remove_uber_projects.php:9]: Should probably be `Removing up_projectid column from projects table`.

File: SETUP/upgrade/14/20200827_remove_uber_projects.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        DROP COLUMN up_projectid
+";
+
+echo "$sql\n";
+
+mysqli_query(DPDatabase::get_connection(), $sql) or die( mysqli_error(DPDatabase::get_connection()));
+
+// ------------------------------------------------------------
+
+echo "Removing uber projects table...\n";
```
Comment (srjfoo) [inline SETUP/upgrade/14/20200827_remove_uber_projects.php:22]: `uber_projects`

**Comment by chrismiceli** [comment]
> In `SETUP/INSTALL.md`, should `-- UberProjects` be removed as a forum to create?

I wasn't sure, I saw some threads were active there referring to related projects, especially on project gutenburg. There were some other references too in some .txt project comments templates that I interpreted the same way.

**Comment by srjfoo** [comment]
> > In `SETUP/INSTALL.md`, should `-- UberProjects` be removed as a forum to create?
> 
> I wasn't sure, I saw some threads were active there referring to related projects, especially on project gutenburg. There were some other references too in some .txt project comments templates that I interpreted the same way.

/me looks again

Good call, and nevermind. That's just giving a list of what we do have defined, it's not the list of suggested forums to create. And no, we don't want to delete that forum.

**Comment by cpeel** [comment]
I want to do a bit of testing with it, but the code changes look good. I want to merge this in after the code release, so please move the upgrade script to a (new) `SETUP/upgrade/15` directory.

**Comment by cpeel** [comment]
> > In `SETUP/INSTALL.md`, should `-- UberProjects` be removed as a forum to create?
> 
> I wasn't sure, I saw some threads were active there referring to related projects, especially on project gutenburg. There were some other references too in some .txt project comments templates that I interpreted the same way.

Please remove the line in `INSTALL.md` but leave it in the template TXT files. Also remove `SETUP/dbdocs/uber_projects.md` (getting validation of these in CI is on my todo list, as is filling out the docs <sigh> so much to do).

**Comment by chrismiceli** [comment]
> uber_projects

uber_projects.md is autogenerated so I don't think there is anything for me to delete from source control. That does get me thinking though if we need to add SETUP/dbdocs to .gitignore so no one accidentally commits it.

**Pull request merged**